### ISSUE TITLE
Clean up database file paths in both Private and Public as per the discussion in clockworklabs/SpacetimeDB#645

### ISSUE THREAD

**Issue by cloutiertyler** [clockworklabs/SpacetimeDB#1385]
See: https://github.com/clockworklabs/SpacetimeDB/pull/645

**Comment by cloutiertyler** [comment]
This ticket should first take the form of a proposal specifying the new path structure. Once that is approved it should be implemented.

**Comment by cloutiertyler** [comment]
Closing in favor of https://github.com/clockworklabs/SpacetimeDBPrivate/pull/850
